smallrye-reactive-messaging-provider/src/test/java/io/smallrye/reactive/messaging/inject/EmitterInjectionTest.java

```diff
@@ -220,16 +380,16 @@ public void run() {
         }
 
         @Incoming("foo")
-        public void consume(String s) {
+        public void consume(final String s) {
             list.add(s);
         }
     }
 
     public static class BeanWithMissingStream {
         @Inject
-        @Stream("missing")
+        @Channel("missing")
```

Comment:
This test is testing the behavior of the legacy @Stream annotation, so don't replace it.

Reply:
done